ako-operator/helm/ako-operator/crds/ako.vmware.com_hostrules.yaml

```diff
@@ -111,6 +111,10 @@ spec:
                       loadBalancerIP:
                         type: string
                     type: object
+                  aliases:
```

Comment:
@RiyaGaniga-avi can you verify these changes?

Reply:
Looks good to me! Can we also add the changes here https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/938f3d29eea194293a1213f5bc5f815d38068237/ako-operator/controllers/crd.go#L36.

Reply:
Added the changes in the ako-operator crd.go file as per the comment.